posthog/api/capture.py

```diff
@@ -54,6 +54,12 @@
 # fewer restrictions on e.g. the order they need to be processed in.
 SESSION_RECORDING_EVENT_NAMES = ("$snapshot", "$performance_event")
 
+EVENTS_RECEIVED_COUNTER = Counter(
+    "capture_events_received_total",
+    "Events received by capture, both session and analytics events.",
```

Comment:
nitpick, to prepare for upcoming data types 🤫 

```suggestion
    "Events received by capture, tagged by resource type.",
```